modules/ROOT/pages/docker-ce.adoc

```diff
@@ -0,0 +1,59 @@
+= Installing Docker CE
+
+By default, Fedora CoreOS comes with out-of-the-box support for `docker` CLI (as provided via https://mobyproject.org/[Moby]). However, in some cases Docker Community Edition (CE) may be preferred for various reasons. This page explains how to replace the provided version with the latest from the upstream Docker sources.
```

Comment:
Styling for the entire change: asciidoc works best with a single sentence per line, each sentence on it's own line.